public/src/data/save.js

```diff
@@ -80,6 +80,15 @@ export function generateSaveData(name) {
     // Getting list of dependencies for each circuit
     for (id in scopeList) { dependencyList[id] = scopeList[id].getDependencies(); }
 
+    // Helper function to get the order of tabs
+    function tabsOrder() {
```

Comment:
Name it as `getTabsOrder()`
And the helper function when loading as `setTabsOrder()`